contracts/LockedTokens.cdc

```diff
@@ -373,10 +373,19 @@ pub contract LockedTokens {
             self.tokenManager = tokenManager
         }
 
+        access(self) fun nodeObjectExists(_ managerRef: &LockedTokenManager): Bool {
+            return managerRef.nodeStaker != nil
+        }
+
         /// Stakes new locked tokens
         pub fun stakeNewTokens(amount: UFix64) {
             let tokenManagerRef = self.tokenManager.borrow()!
 
+            assert (
```

Comment:
Just my OCD: Is there a reason for putting a space inbetween? `assert` is just a function like any other

Reply:
I guess because I kind of view it as a function body instead of an argument list, so I treat it like a curly brace. but you're right that it should just be right after